templates/includes/sidebar.html

```diff
@@ -1,3 +1,10 @@
+{% if user.is_authenticated %}
+  <a href="drivers/{{ user.id }}/">User: {{ user.username }}</a>
+  <a href="{% url "logout" %}">Log Out</a>
+{% else %}
```

Comment:
It will be nice to add <br> between User and Login and Logout. Now it is all in one line